crates/samples/only-bundled-libs/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+//! The known use case for no-default-features is as part of existing apps that already have lifecycle management
+//! in another language
+//! So building a sample app is somewhat more involved
+//! For now, we just check that we can compile mssf-core and mssf-com without any features except for the bundled import libs.
```

Comment:
What is testing? The motivation is unclear.

Reply:
Improved the comment